less/common/sideNav.less

```diff
@@ -65,6 +65,56 @@
       margin-top: 15px;
       border-bottom: 1px solid @control-bg;
     }
+    .sideNav .Dropdown--select .Dropdown-menu>.Dropdown-separator {
+    background:none
+    }
+    
+    .sideNav--horizontal .Dropdown-separator {
+    display:none
+    }
+    
+    .sideNav .Dropdown-separator {
+    display:none
+    }
+    
+    .Dropdown-separator {
+    margin:8px 0;
+    background-color:#e4e8f6;
+    height:1px
+    }
+    
+    @media (min-width: 768px) {
+    .Dropdown--split .Dropdown-menu li:first-child,.Dropdown--split .Dropdown-menu li:first-child+li.Dropdown-separator {
+    display:none
+    }
+ }
+    .Dropdown-separator {
+    margin:0
+    }
+    
+    .TagsPage .sideNav .Dropdown-separator {
```

Comment:
Styling (or code) related to a core extension should never be part of flarum/core.